release/models/system/openconfig-system-logging.yang

```diff
@@ -429,6 +435,22 @@ revision "2023-07-20" {
         "Sets the destination port number for syslog UDP messages to
         the server.  The default for syslog is 514.";
     }
+
+    leaf transport-security {
+      type boolean;
+      description
+      "Indicates if syslog transport layer security (TLS) is enabled.";
+    }
+
+    leaf tls-profile-id {
+      type string;
+      description
+        "The ID of this syslog server's TLS profile.  TLS profiles are managed
```

Comment:
:s/server's/client's/

(the server is a remote destination in this case, and a router is the client)